app/controllers/message_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MessageController < ApplicationController
+  def index
+    @message = Message.order('RANDOM()').first
+    render json: { greeting: @message.greeting }
+  end
```

Comment:
* I get CORS errors in my console when I try to run your app, Kindly [Install the CORS gem and follow the configurations to fix this error](https://github.com/cyu/rack-cors) 
![image](https://github.com/yosaddis/hello-rails-back-end/assets/52497246/41b4d1b5-eef8-4207-bc4a-a5d6bf57a2c8)